### PR TITLE
Remove post-method selection

### PR DESCRIPTION
**Pull request by mets11rap**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "arrow-parens": "error",
     "no-lonely-if": "error",
     "max-len": ["error", {
+      "code": 90,
```
Comment (alex-taxiera) [inline .eslintrc]: lol no

File: src/QueryBuilder.test.js [hunk 2]
```diff
@@ -160,7 +160,10 @@ test('delete', (t) => {
       table,
       where: { key: 'delete this' }
     }))
-    .then((rows) => t.falsy(rows.find((entry) => entry.key === 'delete this')))
+    .then(() => t.context.QueryBuilder.select(table)
+      .then((rows) =>
```
Comment (alex-taxiera) [inline src/QueryBuilder.test.js]: this `then` can be chained on the `then` from before, as long as the previous `then` returns a promise (it does)

File: src/QueryBuilder.test.js [hunk 3]
```diff
@@ -172,9 +175,12 @@ test('delete: whereNot', (t) => {
     .then(() =>
       t.context.QueryBuilder.delete({ table, whereNot: { key: 'dont delete' } })
     )
-    .then((rows) => {
-      t.falsy(rows.find((row) => row.key === 'delete this'))
-      t.truthy(rows.find((row) => row.key === 'dont delete'))
+    .then(() => {
```
Comment (alex-taxiera) [inline src/QueryBuilder.test.js]: drop the curly braces and you can get rid of `return` and can chain the `then` on the previous `then`

File: src/QueryBuilder.test.js [hunk 4]
```diff
@@ -227,9 +233,10 @@ test('increment', (t) => {
       amount: 2,
       where: { key: 'increment this' }
     }))
-    .then((rows) =>
-      t.is(rows.find((row) => row.key === 'increment this').integer, 5)
-    )
+    .then(() => t.context.QueryBuilder.select(table)
+      .then((rows) =>
```
Comment (alex-taxiera) [inline src/QueryBuilder.test.js]: this `then` can be chained on the `then` from before, as long as the previous `then` returns a promise (it does)

File: src/QueryBuilder.test.js [hunk 5]
```diff
@@ -250,16 +257,22 @@ test('increment: whereNot', (t) => {
         key: 'dont increment'
       }
     }))
-    .then((rows) => {
-      t.is(rows.find((row) => row.key === 'increment this').integer, 5)
-      t.is(rows.find((row) => row.key === 'dont increment').integer, 3)
+    .then(() => {
+      return t.context.QueryBuilder.select(table)
```
Comment (alex-taxiera) [inline src/QueryBuilder.test.js]: drop the curly braces and you can get rid of `return` and can chain the `then` on the previous `then`

File: src/QueryBuilder.test.js [hunk 6]
```diff
@@ -250,16 +257,22 @@ test('increment: whereNot', (t) => {
...
+        })
     })
 })
 
 test('insert', (t) => {
   const table = t.context.tables.general
   return t.context.QueryBuilder.insert({ table, data: { key: 'insert this' } })
-    .then((rows) => t.truthy(rows.find((entry) => entry.key === 'insert this')))
+    .then(() => t.context.QueryBuilder.select(table)
+      .then((rows) =>
```
Comment (alex-taxiera) [inline src/QueryBuilder.test.js]: this `then` can be chained on the `then` from before, as long as the previous `then` returns a promise (it does)

File: src/QueryBuilder.test.js [hunk 7]
```diff
@@ -366,9 +379,10 @@ test('update', (t) => {
       where: { key: 'update this' },
       data: { string: 'updated' }
     }))
-    .then((rows) => t.is(
-      rows.find((entry) => entry.key === 'update this').string, 'updated')
-    )
+    .then(() => t.context.QueryBuilder.select(table)
+      .then((rows) => t.is(
```
Comment (alex-taxiera) [inline src/QueryBuilder.test.js]: this `then` can be chained on the `then` from before, as long as the previous `then` returns a promise (it does)

File: src/QueryBuilder.test.js [hunk 8]
```diff
@@ -383,8 +397,11 @@ test('update: whereNot', (t) => {
       whereNot: { key: 'dont update' },
       data: { string: 'updated' }
     }))
-    .then(async (rows) => {
-      t.is(rows.find((entry) => entry.key === 'update this').string, 'updated')
-      t.is(rows.find((entry) => entry.key === 'dont update').string, 'string')
+    .then(() => {
```
Comment (alex-taxiera) [inline src/QueryBuilder.test.js]: drop the curly braces and you can get rid of `return` and can chain the `then` on the previous `then`

File: src/QueryBuilder.test.js (hunk 3 above)
Comment (mets11rap) [reply src/QueryBuilder.test.js]: Not sure what you mean by this.

File: src/QueryBuilder.test.js (hunk 5 above)
Comment (mets11rap) [reply src/QueryBuilder.test.js]: Not sure what you mean by this

File: src/QueryBuilder.test.js (hunk 8 above)
Comment (mets11rap) [reply src/QueryBuilder.test.js]: Not sure what you mean by this

File: .eslintrc (hunk 1 above)
Comment (mets11rap) [reply .eslintrc]: What am I supposed to do with
`return t.is(rows.find((row) => row.key === 'dont increment').integer, 3)`

`t.is(rows.find((entry) => entry.key === 'update this').string, 'updated')`
and
`return t.is(rows.find((entry) => entry.key === 'dont update').string, 'string')`

**Pull request merged**